<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpPrinterGeneralWidget</class>
 <widget class="QWidget" name="TcpPrinterGeneralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>443</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>38</horstretch>
    <verstretch>44</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="deleteCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Delete all IP printer connections</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ipaddressLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>142</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>IP Address:</string>
     </property>
     <property name="buddy">
      <cstring>ipaddressLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="usednsCheckBox">
     <property name="text">
      <string>Use DNS name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="localNameLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>142</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Local name:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="pathLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>91</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Printer path:</string>
     </property>
     <property name="buddy">
      <cstring>pathLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="pathLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QToolButton" name="sharedPathToolButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QCheckBox" name="defaultCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Set this printer as the default printer...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QCheckBox" name="skipLocalCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>... only if a local printer is not present</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="locationLabel">
     <property name="text">
      <string>Location:</string>
     </property>
     <property name="buddy">
      <cstring>locationLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="commentLabel">
     <property name="text">
      <string>Comment:</string>
     </property>
     <property name="buddy">
      <cstring>commentLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="actionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>142</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Action:</string>
       </property>
       <property name="buddy">
        <cstring>actionComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="actionComboBox">
       <property name="currentText">
        <string>Create</string>
       </property>
       <item>
        <property name="text">
         <string>Create</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Replace</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delete</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="ipaddressLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="localNameLineEdit"/>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLineEdit" name="locationLineEdit"/>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QLineEdit" name="commentLineEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>actionComboBox</tabstop>
  <tabstop>deleteCheckBox</tabstop>
  <tabstop>ipaddressLineEdit</tabstop>
  <tabstop>usednsCheckBox</tabstop>
  <tabstop>localNameLineEdit</tabstop>
  <tabstop>pathLineEdit</tabstop>
  <tabstop>sharedPathToolButton</tabstop>
  <tabstop>defaultCheckBox</tabstop>
  <tabstop>skipLocalCheckBox</tabstop>
  <tabstop>locationLineEdit</tabstop>
  <tabstop>commentLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>defaultCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>skipLocalCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
